es-ds-components/components/es-error-page.vue

```diff
@@ -4,7 +4,7 @@ import IconCommunitySolar from './icon/community-solar.vue';
 import IconInfoBlogPost from './icon/info-blog-post.vue';
 import IconMarketplace from './icon/marketplace.vue';
 
-const ES_DOMAIN = process.env.ES_DOMAIN || 'https://www.energysage.com';
+const ES_DOMAIN = 'https://www.energysage.com';
```

Comment:
Should we make this an optional prop instead? Where it still defaults to `https://www.energysage.com` if the prop is not provided? That way the error page can have proper links on int, and the downstream repo can handle getting the appropriate env variable and passing it in.

Reply:
Took me a bit of searching to find it too - it's error.vue in storyblok-shared.